packages/cursorless-org-docs/plugins/node/index.js

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line
+module.exports = function (context, options) {
```

Comment:
This plugging is necessarily to instruct webpack that docusaurus uses to ignore the imports off path and fs that we have in commons.

Reply:
I removed this hack now that all node references have been removed from commons 🎉